make/RunTests.gmk

```diff
@@ -595,8 +595,9 @@ define SetupRunMicroTestBody
     $1_MICRO_BASIC_OPTIONS += -rff $$($1_TEST_RESULTS_DIR)/jmh-result.$(MICRO_RESULTS_FORMAT)
   endif
 
-  # Set library path for native dependencies
-  $1_JMH_JVM_ARGS := -Djava.library.path=$$(TEST_IMAGE_DIR)/micro/native
+  # Set paths for dependencies
```

Comment:
@cl4es So basically this is the line you're objecting to? 

I can remove the change in RunTests.gmk for this patch. That way the wb.jar will be available, but additional work is needed for the micros to access it. But that might, as you say, perhaps be solved within the micros themselves, by annotations or whatever.

Does that sound reasonable?

Reply:
More reasonable, yes. If we need to pack the wb.jar into the benchmarks.jar or can reference it from the test image dir remains to be seen, but it would be the responsibility of those micros that need it to set that up. While a fully self-contained solution might be preferred for simplicity, we can assume that our test harnesses have access to the test image and can pull in the associated wb.jar from there if needed.